_src/static/sass/utilities/_variables.scss

```diff
@@ -362,6 +363,12 @@ $headings-primary-color: palette(grayscale, x-dark);
 $headings-secondary-font-family: $font-family-sans-serif;
 $headings-secondary-color: palette(grayscale, base);
 
+// buttons
+$buttons-border-style: solid;
+$buttons-border-size: 1px;
+$buttons-border-radius: 3px;
+$buttons-shading: 3px;
```

Comment:
This value is really only used in our defined buttons with ``.btn-elevated``. Should this variable be renamed to show that scope (i.e. ``.btn-elevated-shading``)?